lib/services/presets.go

```diff
@@ -316,6 +403,34 @@ func defaultAllowAccessReviewConditions(enterprise bool) map[string]*types.Acces
 func AddRoleDefaults(role types.Role) (types.Role, error) {
 	changed := false
 
+	// Role labels
+	defaultRoleLabels, ok := defaultRoleMetadataLabels()[role.GetName()]
+	if ok {
+		metadata := role.GetMetadata()
+
+		if metadata.Labels == nil {
+			metadata.Labels = map[string]string{}
+		}
+		for label, value := range defaultRoleLabels {
+			if metadata.Labels[label] != value {
+				metadata.Labels[label] = value
+				changed = true
+			}
+		}
```

Comment:
I don't think we should blindly overwrite this. Imagine someone sets it to false, we should probably respect that and not touch the role. Probably we should only set it on creation, or set it if it's unset for OG preset roles (editor,access,auditor)